### DISCUSSION TITLE
Organization url taken

### DISCUSSION THREAD

**Discussion by zrcdn** [community/community#129520] [Enterprise]
### Body

Hi. Our organization url is taken on Github. Empty profile owned by no one knows who https://github.com/paidwork. Is there any contact to get this empty organization url for a company that has a "paidwork" word trademark in every country?

**Comment by manav-gopal** [comment]
To reclaim a GitHub username or organization name that is already taken, especially if it's inactive and you have a trademark for it, you can follow these steps:

1. Check GitHub's Name Squatting Policy
GitHub has a policy against name squatting. If an account is inactive and you're a trademark holder, you might have a valid claim.

2. Contact GitHub Support
You need to contact GitHub Support to address the issue. Here’s how to do it:

### Steps to Contact GitHub Support:
**Go to GitHub Support Page**: Visit [GitHub Support](https://support.github.com/).
**Submit a Ticket**: Select "Contact us" and then choose "Account recovery" or "General account question" depending on the available options.
**Provide Details**: In your message, include the following information:
Your organization’s trademark details (registration number, countries where the trademark is registered).

The URL of the inactive GitHub organization (https://github.com/paidwork).
A statement explaining your trademark ownership and why you need the specific GitHub username/organization name.
Any evidence of your attempts to contact the current owner, if any.

3. Provide Trademark Documentation
Prepare to provide official documentation proving your trademark ownership if GitHub Support requests it.

Example Message to GitHub Support

```text
Subject: Request for Organization Name Reassignment - Trademark Holder

Hi GitHub Support Team,

I hope this message finds you well. I am writing on behalf of [Your Company Name], which owns the trademark for "PaidWork" in [list of countries]. Our official trademark registration number is [registration number], and we have registered this trademark in [list of countries].

We have noticed that the GitHub organization name "paidwork" (https://github.com/paidwork) is currently taken by an account that appears to be inactive. This organization is not associated with our company, and the inactivity suggests that it may be name squatting.

As the legitimate trademark holder for "PaidWork," we kindly request the reassignment of the "paidwork" organization name to our company. We believe this change will help us avoid any confusion and ensure that our customers can easily find our official GitHub repository.

Please let us know if you require any further information or documentation to process this request. We appreciate your assistance and look forward to your response.

Best regards,

[Your Full Name]
[Your Position]
[Your Contact Information]
[Your Company Name]
```
4. Wait for Response
Be patient while waiting for a response. GitHub Support may take some time to investigate and verify your claim.

5. Follow Up if Necessary
If you do not receive a response within a reasonable timeframe (e.g., a couple of weeks), consider following up with GitHub Support to check on the status of your request.

By following these steps, you should be able to present a strong case to GitHub for reclaiming the organization name "paidwork" based on your trademark rights.

**Comment by davevad93** [comment]
Hi @zrcdn ,

Read GitHub's documentation about [GitHub Trademark Policy](https://docs.github.com/en/site-policy/content-removal-policies/github-trademark-policy) that clearly establishes what is or isn't considered an infringement of trademark policies. You can submit a report here:

- [Submit a Trademark Policy Violation Report](https://support.github.com/contact/trademark-policy)
